smoke/IotEdgeQuickstart/details/Details.cs

```diff
@@ -334,6 +339,45 @@ protected async Task VerifyDataOnIoTHub(string moduleId)
             await eventHubClient.CloseAsync();
         }
 
+        protected async Task VerifyTwinAsync()
+        {
+            if (this.TwinTestFileName.HasValue)
```

Comment:
this.TwinTestFileName.Map

Reply:
Cool. Done with ForEachAsync. Thanks for showing!